contracts/GolemNetworkTokenBatching.sol

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 Golem Factory
+// Licensed under the GNU General Public License v3. See the LICENSE file.
+
+pragma solidity ^0.4.19;
+
+import "./ExtendedToken.sol";
+import "./TokenProxy.sol";
+
+contract GolemNetworkTokenBatching is TokenProxy, ExtendedToken {
+
+    string public constant name = "Golem Network Token Batching";
```

Comment:
I dunno we should focus on `batching` in the name, this token provides full ERC20 compliance and ERC677-like functionality as well expect for the `batchTransfer`.
I'm fine with the code and we can agree on the name later if needed.